test/unit/node/generated-code-dependencies.js

```diff
@@ -347,17 +347,26 @@ suite('context test', function() {
     });
   });
 
+  function checkFile(outDir, goldenDir, filename) {
+    var actualContents = fs.readFileSync(path.resolve(outDir, filename));
+    var actual = actualContents.toString().split('\n');
```

Comment:
Hm, I was sure I had seen mocha use string diffs for the error message. I guess that might only be for some reporters.

Keep the split then.